users/updates/components/69qa8h6E1bzG.php

```diff
@@ -5,7 +5,7 @@
 //Release Date Unknown
 //Rewrote 2019-02-19 BA
 
-$db->query("UPDATE logs SET logtype = ? WHERE logtype = ? AND lognote LIKE ?",array("IP Logging","User","%blacklisted%attempted%visit"));
+$db->query("UPDATE logs SET logtype = ? WHERE logtype = ? AND lognote LIKE ?", ["IP Logging","User","%blacklisted%attempted%visit"]);
```

Comment:
What's wrong with the other syntax?

Reply:
Actually this was the previous syntax which i changed but after checking, both were working so I reverted to the previous one. I can revert if you want?

Reply:
Please always compare to the [original version](https://github.com/mudmin/UserSpice5/blob/master/users/updates/components/69qa8h6E1bzG.php) when in doubt.

Reply:
Ok, my bad I will revert it, I did some changes like this before, so I thought this was one of them. Anyway, thanks for letting me know.